linux_os/guide/system/var_remote_paths.var

```diff
@@ -0,0 +1,16 @@
+documentation_complete: true
+
+title: 'Explicit remote paths'
+
+description: |-
+    Colon-separated list of paths that are remote mounts s.a. network filesystems, and that are intended to be filtered out from scanning.
+    In some cases, the scanner may not detect local filesystems correctly, and this variable is meant to be used as a user-friendly way to designate paths as non-local.
+    This variable is supposed to be used in rules that use the concept of local filesystems in their checks.
+
+type: string
+
+interactive: false
+
+options:
+    default: "/mnt:/net"
```

Comment:
I understand your concerns - while it is a "mostly good default", it may not be appropriate for everybody, which is a deal breaker. 
Note: We will need to enhance the test suite in order to be able to test this rule.